clang/unittests/Format/FormatTestCSharp.cpp

```diff
@@ -1304,6 +1304,18 @@ TEST_F(FormatTestCSharp, CSharpGenericTypeConstraints) {
                "}",
                Style);
 
+  // When the where line is not to be formatted, following lines should not take
```

Comment:
```suggestion
  // When the "where line" is not to be formatted, following lines should not take
```

Maybe so? It took me some time to parse this sentence...